performance/tests/sandstone/ListItemsTests.js

```diff
@@ -13,9 +13,11 @@ const listItemTests = (componentName, dataSize) => describe(componentName, () =>
 	const pageURL = dataSize ? `http://${serverAddr}/${componentName}?dataSize=${dataSize}` : `http://${serverAddr}/${componentName}`;
 
 	describe('ScrollButton', () => {
-		it('scrolls down', async () => {
+		it('scrolls down with native scrollMode', async () => {
+			const pageURLNative = pageURL + '?scrollMode=native';
```

Comment:
scrollMode is shown as undefined when dataSize is set. You should use "&" at the beggining in that case